client/__tests__/e2e/test_config.yaml

```diff
@@ -4,6 +4,14 @@ server:
     port: 5005
     debug: false
     verbose: false
+    disable_auth: true
```

Comment:
Explicitly disable_auth for the standard test server. Combined with base_url=d, this keeps annotation/geneset writes disabled for the main E2Es.